sale_financial_risk/views/res_partner_view.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2016-2018 Tecnativa - Carlos Dauden
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="res_partner_view_risk" model="ir.ui.view">
+        <field name="name">res.partner.view.risk</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="account_financial_risk.res_partner_view_risk" />
+        <field name="arch" type="xml">
+            <field name="risk_invoice_draft_include" position="before">
+                <field
+                    name="risk_sale_order_include"
+                    attrs="{'readonly': [('risk_allow_edit', '=', False)]}"
+                />
+                <button
+                    name="open_risk_pivot_info"
+                    type="object"
+                    class="oe_link"
```

Comment:
Nitpick: `oe_link` deprecated in favor of `btn-link` [[source](https://github.com/odoo/odoo/blob/4a8239c86e3fdf2830bd4f52d2424fe0f83d4639/addons/web/static/src/js/core/dom.js#L405)]